hack/test-prerequisites.go

```diff
@@ -55,5 +59,29 @@ func main() {
 		if err != nil {
 			log.Fatalf("Could not install %s CRD: %v", name, err)
 		}
+		err = wait.PollImmediate(time.Second, 30*time.Second, func() (bool, error) {
+			_, err = cvClient.ConfigV1().ClusterVersions().Create(ctx,
```

Comment:
I think we want to move this creation out of this file, and into a `pkg/start/start_integration_test.go`.  I don't think we need the poll wrapper, unless we feel like we need that for robustness in the face of a flaky Kube-API server connection or some such.  It would slot into each test-case right after namespace creation, around [here][1], and similarly for each of the other `TestIntegrationCVO_...` functions.  Something like:

```go
ns := fmt.Sprintf("e2e-cvo-%s", randutil.String(4))

if _, err := kc.CoreV1().Namespaces().Create(
	ctx,
	&v1.Namespace{
		ObjectMeta: metav1.ObjectMeta{
			Name: ns,
		},
	},
	metav1.CreateOptions{},
); err != nil {
	t.Fatal(err)
}
defer func() {
	if err := kc.CoreV1().Namespaces().Delete(ctx, ns, metav1.DeleteOptions{}); err != nil {
		t.Logf("failed to delete namespace %s: %v", ns, err)
	}
}()

if _, err := client.ConfigV1().ClusterVersions().Create(
	ctx,
	&configv1.ClusterVersion{
		ObjectMeta: metav1.ObjectMeta{
			Name: ns,
		},
		Spec: configv1.ClusterVersionSpec{
			Channel: "fast",
		},
	},
	metav1.CreateOptions{},
); err != nil {
	t.Fatal(err)
}
defer func() {
	if err := client.ConfigV1().ClusterVersions().Delete(ctx, ns, metav1.DeleteOptions{}); err != nil {
		t.Logf("failed to delete cluster version %s: %v", ns, err)
	}
}()
```

It's a bit long-winded, but writing a helper to wrap what's basically two function calls seems fairly complicated too.  And having the two calls inline within `TestIntegrationCVO_...` does make for the convenient `defer` teardown.

[1]: https://github.com/openshift/cluster-version-operator/blob/addb685a2e9f6375ba4ccd3dbfb9f9236475494a/pkg/start/start_integration_test.go#L252